js/index.js

```diff
@@ -1 +1,77 @@
 // Your code goes here
+
+funBusHead = document.querySelector('.logo-heading');
+navBar = document.querySelector('.nav');
+navItems = document.querySelectorAll('.nav-link');
+headerH2 = document.querySelector('.intro h2');
+allParagraphs = document.querySelectorAll('p');
+allExpeditionsTitle = document.querySelectorAll('.destination h4');
+footer = document.querySelector('.footer');
+
+funBusHeading.addEventListener('mouseover', function () {
+    this.classList.add('funBusHeading');
+});
+
+funBusHead.addEventListener('mouseleave', function () {
+    this.class.remove('funBusHeading');
```

Comment:
What this code says is:
When the mouse leaves the `.logo-heading` div, this function will run. 

This function tries to remove the class `funBusHeading` from `.logo-heading`, but no such class exists in your less files or anywhere in your index.html.

So this will error out. Review this eventlistener and see if you can figure out how to get it to work. 

Please contact me if you have any questions about this.